src/explore-education-statistics-admin/src/pages/release/data/utils/getUnmappedLocationErrors.ts

```diff
@@ -1,23 +1,21 @@
-import { UnmappedAndManuallyMappedLocation } from '@admin/pages/release/data/utils/getApiDataSetLocationMappings';
+import { MappableLocation } from '@admin/pages/release/data/utils/getApiDataSetLocationMappings';
 import { ErrorSummaryMessage } from '@common/components/ErrorSummary';
 import { Dictionary } from '@common/types';
 import locationLevelsMap from '@common/utils/locationLevelsMap';
 import camelCase from 'lodash/camelCase';
 
 export default function getUnmappedLocationErrors(
-  unmappedAndManuallyMappedLocations: Dictionary<
-    UnmappedAndManuallyMappedLocation[]
-  >,
+  mappableLocations: Dictionary<MappableLocation[]>,
```

Comment:
As elsewhere - can update the type of this to `Record<LocationLevelKey, MappableLocation[]>`